### ISSUE TITLE
Cannot add climate to climate

### ISSUE THREAD

**Issue by gvhomeassistant** [Blackymas/NSPanel_HA_Blueprint#1104]
Hi 
In climate page, I cannot add other Climate devices as Entity 

I am using 4.0.1 of TFT and NSPanel

**Comment by Doughboy68** [comment]
The purpose of those buttons is not to add other climate entities, but possibly to have entities to enhance function of  current thermostat... maybe a fan, or something else. Not sure really what the original idea behind these were, maybe can be explained.

The sensors howeverI use those with four ecobee remote temp sensors around the house.

The only way to add another climate device is either use a button in a button page and long press, or one of the three homepage custom buttons (which I use on one of mine to control fireplace).

**Comment by edwardtfn** [comment]
The problem is to open a detailed page from a detailed page...
When I say "detailed page" I mean one of these:
- Alarm (limited for now)
- Climate
- Cover
- Fan
- Light
- Media Player

Usually those pages will be opened by a long press in a button (button pages or hardware buttons) or from the custom buttons in Home page.
When calling those pages, the system sets two global vars to retain the `entity_id`, used to all the service calls, and the "back page", or the page from where the detailed page was opened, so the system know where to return to when the detailed page is closed.
The system offers several limitations to have recursive calls and to store information about other instances of the Blueprint, so we have to rely on the limited global memories in ESPHome or TFT.
In short, we would have an issue opening a detailed page from a detailed page as the entity_id and previous page of the first one would be lost when that calls a second one (and nothing will prevents users to go deeper and deeper infinitely open another detailed paga from a detailed page). This would add lots of complexity and, to avoid that, we simply removed that option.
It could be possible to have a climate entity as a button on the climate page, but that would be a toggle button only (as it is for lights, etc)... Will that be any usefull?

**Comment by gvhomeassistant** [comment]
I understand, and you are right, it would not be usefull :)
